src/api/queries/getMetricQueries.js

```diff
@@ -0,0 +1,55 @@
+var timeBound = ['', '1d', '7d', '30d', '60d', '90d', '180d', '365d', '2y', '3y', '5y', '10y']
```

Comment:
global var allow mutation from everywhere? 
please use `const TIME_BOUND_VALUES` or something like that

Reply:
The javascript is using ECMA5, which doesn't include const or var, I will rename the variables, but cannot use anything else except for 'var'